modules/backup-and-restore/pages/configurations.adoc

```diff
@@ -34,7 +35,9 @@ Required if backup is to be stored on S3.
 |`+nan+`
 
 |System.Backup.S3.BucketName |Name of the S3 bucket to store backup files.
-Required if backup is to be stored on S3.|`+nan+`
+Required if backup is to be stored on S3.|null
+
+|System.Backup.S3.Endpoint |Storage URL (fully qualified) to use instead of default endpoint for S3 client, typically used when operating in a private network or with a non-AWS S3-compatible endpoint. |null
```

Comment:
```suggestion
|System.Backup.S3.Endpoint | Endpoint to use instead of the S3 default endpoint. Must be a fully qualified URL.
Typically used when operating in a private network or with a non-AWS S3-compatible endpoint. | null
```